next-tavla/src/Shared/components/Header/styles.module.css

```diff
@@ -1,6 +1,7 @@
 .logo {
     height: auto;
     width: 7em;
+    min-width: 250px;
 }
 
```

Comment:
Ken redusere min-width, logoen føles veldig stor på admin og landing. I figma er det ish 185px

Reply:
Kan vurdere å lage egen heading-komponent for board, hvis det blir knot med font størrelse og em-units